src/core/QCheck.mli

```diff
@@ -886,120 +909,16 @@ module Test : sig
         (since 0.10)
   *)
 
-  val get_arbitrary : 'a cell -> 'a arbitrary
-  val get_law : 'a cell -> ('a -> bool)
-  val get_name : _ cell -> string
-  val set_name : _ cell -> string -> unit
-
-  val get_count : _ cell -> int
-  (** Get the count of a cell.
-      @since 0.5.3 *)
-
-  val get_long_factor : _ cell -> int
```

Comment:
No, it's moved to QCheck2, just like the `Test.cell` and `Test.t`

Reply:
the other ones seem to still be there, but forwarded to QCheck2, aren't they?

Reply:
I don't think so: `QCheck.Test` only contains:
* fail_report
* fail_reportf
* make_cell
* set_name
* make

All cell getters like `get_arbitrary` are only in QCheck2

Reply:
I have put back (in 42c8c23b61ccaae051b23c2803298eaa67f6d2e4):
* get_name
* get_count
* get_law
* get_long_factor

I have not put back `get_arbitrary` as it would return a QCheck2 arbitrary rather than a QCheck1 arbitrary, which I think doesn't make much sense

Is that what you had in mind?

Reply:
that'll do, yes :)